radio/src/lua/api_model.cpp

```diff
@@ -1586,11 +1586,68 @@ static int luaModelResetSensor(lua_State *L)
   if (idx < MAX_TELEMETRY_SENSORS) {
     telemetryItems[idx].clear();
   }
-  
+
   lua_pushnil(L);
   return 1;
 }
+/*luadoc
+@function model.swashRingData(swash, line, value)
+
+Set heli swash parameters
+
+@param swash (unsigned number) number (use 0)
```

Comment:
If it needs to be 0, does it make sense to ask inputing it ?

Reply:
No, but it has to be in the command otherwise it will throw an error when executing it, a similar situation with the line number, so there has to be an explanation.

Would something like
@param swash  must be zero "0"

be acceptable?

Reply:
language changed to required
![Capture](https://user-images.githubusercontent.com/20526716/125503092-cf11dae4-e3a8-400d-8958-07b7530eda7e.PNG)